DOCKER_README.md

```diff
@@ -0,0 +1,53 @@
+## Docker Compose User Guide
```

Comment:
Awesome thanks for all the detailed instructions! Doing a brief code review now, will dig into this more ASAP